client/app/components/pages/BamView.vue

```diff
@@ -1,1179 +1,1526 @@
-      },
-
-      receiveClinMessage: function(event) {
-        let self = this;
-        // Do we trust the sender of this message?
-        if (this.clinIobioUrls.indexOf(event.origin) == -1) {
-          return;
-        }
-        this.clinIobioUrl = event.origin;
-
-        var clinObject = JSON.parse(event.data);
-
-        if (clinObject.type == 'show-tooltip') {
-          self.clinTooltip[clinObject.task.key].show = true;
-          self.clinTooltip[clinObject.task.key].content = clinObject.task.tooltip;
-        } else if (clinObject.type == 'hide-tooltip') {
-          self.clinTooltip[clinObject.task.key].show = false;
-        }
-
-      }
-    },
-
-    created: function () {
-      this.load();
-    },
-
-    mounted: function() {
-      let self = this;
-      window.addEventListener("message", self.receiveClinMessage, false);
-    },
-
-    watch: {
-      readOutliers: function() {
-        this.updateLengthHistograms();
-      },
-    },
-
-    computed: {
-      pathToFilename: function(path) {
-        if (this.selectedBamURL !== undefined) {
-          const pathParts = this.selectedBamURL.split('/');
-          const filename = pathParts[pathParts.length - 1];
-          return filename;
-        }
-
-        return '';
-      },
-      // TODO: Determine if this is needed
-      //shortenedBamFileURL: function() {
-      //  if (this.selectedBamURL !== undefined) {
-      //    const protocolDiv = this.selectedBamURL.split('//');
-
-      //    const protocol = protocolDiv[0];
-      //    const withoutProtocol = protocolDiv[1];
-
-      //    let pathDivision = withoutProtocol.split('/');
-
-      //    // Edge case for '/' ending causing empty item
-      //    if (pathDivision[pathDivision.length - 1] === '') {
-      //      pathDivision.splice(pathDivision.length - 1, 1);
-      //    }
-
-      //    const host = pathDivision.shift();
-      //    const fileName = pathDivision.pop();
-      //    const middlePart = pathDivision.join("/");
-
-      //    const fontSize = 25;
-      //    const sizingStr = `<span style="font-size: ${fontSize}pt"></span>`;
-
-      //    if (middlePart.length === 0) {
-      //      return sizingStr + this.selectedBamURL;
-      //    }
-      //    else if (this.showFullURL) {
-      //      return protocol + "//" + host +
-      //        sizingStr + "/" + middlePart + "/" +
-      //        (fileName !== undefined ? fileName : "")
-      //    }
-      //    else {
-      //      return protocol + "//" + host +
-      //        `/<span style="font-size: ${fontSize}pt;color:#2687BE;" >...</span> /` +
-      //        (fileName !== undefined ? fileName : "")
-      //    }
-      //  }
-      //  return '';
-      //},
-    }
-
-  }
-
-
-  function precisionRound(number, precision) {
-    var factor = Math.pow(10, precision);
-    return Math.round(number * factor) / factor;
-  }
-
+import ReadsSampledBox from '../partials/ReadsSampledBox.vue';
```

Comment:
Seeing this, there's got to be a vue congifs/plugin to cover these details. an empty line between content types should be respected

```suggestion

import ReadsSampledBox from '../partials/ReadsSampledBox.vue';
```